ebib-utils.el

```diff
@@ -2387,14 +2387,17 @@ are returned as a string."
 (defun ebib-display-www-link (field key db)
   "Return the content of FIELD from KEY in DB as a link.
 This function is mainly intended for the DOI and URL fields."
-  (let ((str (ebib-get-field-value field key db 'noerror 'unbraced 'xref)))
-    (if (and str (string-match-p "^[0-9]" str))
-        (setq str (concat "https://dx.doi.org/" str)))
-    (if str
-        (propertize "www"
-                    'face '(:height 0.8 :inherit ebib-link-face)
-                    'mouse-face 'highlight
-                    'help-echo str)
+  (let* ((val (ebib-get-field-value field key db 'noerror 'unbraced 'xref))
+	 ;; Unless field is doi, assume the string is a full url
+	 (str (if (and (case-fold-string= field "doi")
```

Comment:
`case-fold-string=` isn't part of Emacs. It's actually given as an example in the Emacs manual `(info "(elisp) Defining Hash")`, perhaps you got it from there?

Reply:
Strange -- I checked and it actually comes from EMMS (which I use, so I have it loaded). EMMS doesn't prefix the function with a proper name, so I assumes it was part of Emacs itself. Fixed.